### ISSUE TITLE
Issue with Pyuploadcare - Image Upload option is hidden instead it should be "file"

### ISSUE THREAD

**Issue by bikiransv** [bikirandas/OmexOnline#5]
Using Pyuploadcare module for cover pic selection feature so that the Image upload can be more attractive with cropping and drag-drop features 

form generates the following html code:

```
<input type="file" name="img" accept="image/*" id="id_img"></td></tr>
<tr><th><label for="id_cover">Cover:</label></th><td><input type="hidden" name="cover" role="uploadcare-uploader" data-public-key="xxxxxxxxxxxxxxxxxxxxxx" data-integration="Django/2.1.7; PyUploadcare-Django/2.6.0" data-clearable="" data-images-only="" id="id_cover"><
/td></tr>
```

**Issues Encountered after Pyuploadcare implementation**

1. Cover pic  Upload option is not visible from the website ProfileEdit page, where as the options is working from the admin interface.
  **Solutions:**       1.  _Update the rendered profile edit form from Accounts/views.py_   

In the above mentioned html code that is generated from the forms.py the type for input with id = "id_cover" is hidden which needs to be file.

_Originally posted by @bikiransv in https://github.com/bikirandas/OmexOnline/issues/3#issuecomment-476993784_

**Comment by bikiransv** [comment]
Check for way to update the form generated in views.py ---> So that the views.py renders the custom designed form instead of the Django generated form.

**Comment by bikiransv** [comment]
Issue Resolved - With the commit 06351c5

Suggestions collected from the below stackoverflow link 👍 
[https://stackoverflow.com/questions/55414672/pyuploadcare-imagefield-upload-option-is-not-visible-at-useredit-html-page/55415612#55415612](https://stackoverflow.com/questions/55414672/pyuploadcare-imagefield-upload-option-is-not-visible-at-useredit-html-page/55415612#55415612)
